### ISSUE TITLE
[BUG] Accessibility Issue with attribution label

### ISSUE THREAD

**Issue by wolfgangzwiauer** [tiagozip/cap#124]
I would love to keep the attribution label visible, unfortunately our accessibility tests are failing.

Our Accessibility Checker throws two errors:
1. Interactive controls must not be nested
<img width="926" height="478" alt="Image" src="https://github.com/user-attachments/assets/157f9d48-de57-406e-9517-49eb968827dc" />

2. All touch targets must be 24px large, or leave sufficient space
<img width="936" height="535" alt="Image" src="https://github.com/user-attachments/assets/3a6efcb7-b176-4cc5-8acc-6aeef66b40e9" />

Hiding the label solves both issues of course.

**Comment by tiagozip** [comment]
hi there, unfortunately for the 2nd one i shouldnt be able to do much since the attribution label touch area is intentionally small to avoid misclicks when someone intends to click the solve button. it shouldn't really affect much since it's big enough for most stuff anyways.

i'll look into the first one.

**Comment by tiagozip** [comment]
i'll be closing this for now, if you have any further questions feel free to ask.

**Comment by wolfgangzwiauer** [comment]
Thanks, we were able to solve it by disabling the label and adding our own, outside of the container, overlaying it via `position: absolute`.
